src/worker.js

```diff
@@ -54,6 +54,10 @@ const initServer = (app, config, callback) => {
       .catch(exit(code));
   }
 
+  function readyToClose(getClose) {
+    getClose(shutDownSequence);
```

Comment:
@goatslacker the intent here is to be able to close down the hypernova server (and transitively the express & http server) from the consuming application on demand.

This is to set us up to have better end to end testing - currently we can only spin up one instance of hypernova per process (i.e. per test run). (more context linked in the body of PR)